content/ja/docs/tasks/debug/debug-application/get-shell-running-container.md

```diff
@@ -47,7 +47,7 @@ kubectl exec --stdin --tty shell-demo -- /bin/bash
 ```
 {{< note >}}
 
-ダブルダッシュの記号 `--` はコマンドに渡す引数とkubectlの引数を分離します。
+ダブルダッシュ(`--`)のセパレーターは、コマンドに渡す引数とkubectlの引数を分離します。
 
 {{< /note >}}
```

Comment:
It doesn't affect the display, but could you delete the blank lines on lines 49 and 51?

Reply:
> It doesn't affect the display, but could you delete the blank lines on lines 49 and 51?

Hi @t-inu , I have removed the blank lines at 49 and 51. Is there any change required?